ha/k8s_setup/ha_start.py

```diff
@@ -13,7 +15,8 @@
                                     )
 
 my_parser.add_argument('--services', '-s', help='service_name', required=True, action='store')
-my_parser.add_argument('--config', '-c', help='config file', required=False, action='store')
+my_parser.add_argument('--config', '-c', help='config file url', required=False, action='store')
+my_parser.add_argument('--debug', '-d', help='debug level {0|1}', type=int, nargs='?', const=1, default=0, required=False, action='store')
```

Comment:
Why need to give debug option here? Don't you think AUTO_RESTART will be sufficient?
Or One point here, that restart should be configurable. In production, we can not afford restart again. But, then only debug is sufficient. Why need both debug and AUTO_RESTART?

Reply:
the purpose is 'debug' not auto restart, auto restart is the solution so debug can make easy.
so if you provide --debug it will enable the auto-restart of the driver process so we can change/update the code easily and test it.
for more details you can read it here: https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/956891431/Debug+option+for+cortx-ha
So auto-restart is not an option it is the internal solution to enable change/update code and test it without restarting pod and no need to commit docker image.

And here I have given support for debug level 1 only tomorrow debug level 2 can do some addition work like enabling debug logging and more.